pyramid_oereb/lib/sources/general_information.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from pyramid_oereb.lib.sources import Base
+from pyramid_oereb.lib.records.general_information import GeneralInformationRecord
+
+
+class GeneralInformationBaseSource(Base):
+    """
+    Base class for GeneralInformation values source.
+
+    Attributes:
+        records (list of pyramid_oereb.lib.records.document_types.GeneralInformationRecord): List of general
```

Comment:
document_types pops up again :)

Reply:
Thanks ! I'm caught, I've been copy-pasting your PR :)

Reply:
it's fair game - I did it before on your 'theme' PR 😄